.taskcluster.yml

```diff
@@ -56,6 +56,36 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("build_documentation")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
+               cd rust-code-analysis &&
+               git -c advice.detachedHead=false checkout ${head_rev} &&
+               cargo install mdbook --no-default-features --features search,output --vers \"^0.1.0\" &&
```

Comment:
What are the default features?

Reply:
I just followed the `mdbook' [README](https://github.com/rust-lang/mdBook#what-does-it-look-like) section called `Note for automatic deployment`

Reply:
:+1: